bin/upload-schema.js

```diff
@@ -7,39 +7,39 @@ auto pushing schema's tricky. For now this script does the uploads manually.
 */
 
 var aws = require('aws-sdk-promise');
+var fs = require('fs');
 
-var loadConfig = require('taskcluster-base/config');
-var config = loadConfig({
-  defaults: require('../config/defaults'),
-  profile: require('../config/production'),
-  filename: 'docker-worker'
+var base = require('taskcluster-base');
+var config = base.config({
+  files: [`${__dirname}/../config.yml`],
```

Comment:
Is this line actually necessary? I wouldn't be surprised if you said yes because of some sort of weird thing with the worker, but I haven't seen us needing to use it before.